runner/runner.sln

```diff
@@ -0,0 +1,23 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.539
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{888888A0-9F3D-457C-B088-3A5042F75D52}") = "runner", "runner.pyproj", "{1BBC824F-E02F-4E05-AF63-615CDA90775D}"
```

Comment:
why is this file added?

Reply:
As in comment for commit, so project can be opened in visual studio. Much easier to manage python environments, install requirements etc

Reply:
Fair enough, just use pycharm since that gives you all the pep8 stuff